tests/testthat/test_TaskClassif_melanoma.R

```diff
@@ -0,0 +1,20 @@
+skip_on_cran()
+
+test_that("melanoma task works", {
+  withr::local_options(mlr3torch.cache = TRUE)
```

Comment:
you also need to run `skip_if_not_installed("curl")` here, because it is only in suggests